src/main/java/uk/gov/ons/census/action/model/dto/FieldworkFollowup.java

```diff
@@ -30,4 +30,5 @@
   private String surveyName;
   private Boolean undeliveredAsAddress;
   private Boolean blankQreReturned;
+  private boolean handDelivery;
```

Comment:
Might be worth making this a `Boolean` instead of a `boolean` if FWMT are going to get fussy about us sending them zero or false, instead of null